torch/csrc/jit/passes/fold_conv_bn.cpp

```diff
@@ -113,8 +113,8 @@ class FoldConvBatchNormHelper {
 
   std::unordered_map<ModulePtr, std::tuple<at::Tensor, at::Tensor>>
       conv_module_and_params_;
-  std::unordered_map<Graph*, std::vector<std::tuple<std::string, std::string>>>
-      conv_bn_names_;
+  std::unordered_map<Graph*, std::vector<std::tuple<std::vector<std::string>, std::vector<std::string>>>>
```

Comment:
this is getting a bit harder to read, perhaps we can define a type for this and/or add a comment describing what this contains?  Otherwise the reader has to read the code line by line.

Reply:
will do